src/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @parent_permission && is_of_age?
```

Comment:
- [ ] Consider that method returns true if a person is of age or if they have permission from parents not and.
It will be like this:
```ruby
  def can_use_services? 
    is_of_age?() || @parent_permission
  end
```